gcc/rust/backend/rust-compile-intrinsic.cc

```diff
@@ -717,7 +746,9 @@ copy_nonoverlapping_handler (Context *ctx, TyTy::FnType *fntype)
     = build2 (MULT_EXPR, size_type_node, TYPE_SIZE_UNIT (param_type), count);
 
   tree memcpy_raw = nullptr;
-  BuiltinsContext::get ().lookup_simple_builtin ("memcpy", &memcpy_raw);
+  BuiltinsContext::get ().lookup_simple_builtin (overlaps ? "memmove"
+							  : "memcpy",
+						 &memcpy_raw);
```

Comment:
```suggestion
						 &memcpy_raw);
```

nit but this variable should probably be renamed since it can be either `memmove` or `memcpy` :)